core/src/main/java/org/sonarsource/sonarlint/core/languages/LanguageSupportRepository.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.languages;
+
+import java.util.Collection;
+import java.util.EnumSet;
+import java.util.Set;
+import org.sonarsource.sonarlint.core.commons.Language;
+
+public class LanguageSupportRepository {
+  private static final EnumSet<Language> LANGUAGES_RAISING_TAINT_VULNERABILITIES = EnumSet.of(Language.CS, Language.JAVA, Language.JS, Language.TS, Language.PHP, Language.PYTHON);
```

Comment:
Quick question, how do you make sure this list is up-to-date?

Reply:
That's a good question! It's pretty rare that a new language is supported in the security analyzer, so I don't expect this list to change frequently